doctrine/doctrine-bundle/1.6/manifest.json

```diff
@@ -8,9 +8,9 @@
     },
     "env": {
         "#1": "Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url",
-        "#2": "For an SQLite database, use: \"sqlite:///%kernel.project_dir%/var/data.db\"",
+        "#2": "For an SQLite database, use: \"sqlite://%kernel.project_dir%/var/data.db\"",
         "#3": "For a PostgreSQL database, use: \"postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=11\"",
-        "#4": "IMPORTANT: You MUST also configure your db driver and server_version in config/packages/doctrine.yaml",
-        "DATABASE_URL": "mysql://db_user:db_password@127.0.0.1:3306/db_name"
+        "#4": "IMPORTANT: You MUST configure your db driver and server version, either here or in config/packages/doctrine.yaml",
+        "DATABASE_URL": "mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7&charset=utf8mb4&defaultTableOptions[collate]=utf8mb4_unicode_ci"
```

Comment:
It becomes a bit unreadable like this.

I'd suggest to keep table options & charset directly in the `doctrine.yaml` file, since it's not likely to change because it's also a project-based option, not machine-based.

By the way, having `utf8mb4_unicode_ci` in an environment and something else in another env might lead to problems like indexes length issue, that people almost always struggle to fix.

Reply:
thus, Saas hosting are very unlikely to provide such query parameters in their DATABASE_URL (even the serverVersion is probably not there)

Reply:
Works for me, updated.